pkgs/os-specific/linux/ena/default.nix

```diff
@@ -43,5 +43,6 @@ stdenv.mkDerivation rec {
     license = licenses.gpl2Only;
     maintainers = with maintainers; [ eelco sielicki ];
     platforms = platforms.linux;
+    priority = -1;
```

Comment:
Why is that?

Reply:
forgot the other line that's a part of this, whoops

```
$ find /nix/store/milx8ikc3qh841f56l669ndqycidf0dz-ena-2.9.1-6.4.11 | grep '/ena.ko' | cut -d'-' -f2-
ena-2.9.1-6.4.11/lib/modules/6.4.11/misc/ena.ko.xz
$ find /nix/store/6zfnb5w30jy06qx6w0hr9zyr7rwzalz6-linux-6.4.11/    | grep '/ena.ko' | cut -d'-' -f2-
linux-6.4.11/lib/modules/6.4.11/kernel/drivers/net/ethernet/amazon/ena/ena.ko.xz
```

The idea is to make it intentionally collide with the ko that (potentially) is provided by the kernel, then set an aggressive priority so that the out of tree module is preferred.

amended the commit to fix the behavior:

```
$ find /nix/store/c51h1j864cp1ddhp144dd2smxw4vyv6d-ena-2.9.1-6.4.11 | grep '/ena.ko' | cut -d'-' -f2-
ena-2.9.1-6.4.11/lib/modules/6.4.11/kernel/drivers/net/ethernet/amazon/ena/ena.ko.xz
```